### PR TITLE
add scram integration

### PR DESCRIPTION
**Pull request by jchomat**
allow connection to kafka with scram 256 or 512

### CODE REVIEW

**Comment by weeco** [comment]
Hey @jchomat ,
thank you for your pull request. I'll try to review it during the holidays somewhere between christmas eve and new years eve.

**Comment by jchomat** [comment]
Hey @weeco, 
Thank you, i 'm using a local binary for now but it will be easier to use for the community;) 
I use it on 2-3 kafka cluster with scram 256 since few days. tls must be enabled to use it ;)

File: options/options.go [hunk 1]
```diff
@@ -43,6 +44,7 @@ type Options struct {
 	TLSCertFilePath          string   `envconfig:"KAFKA_TLS_CERT_FILE_PATH"`
 	TLSInsecureSkipTLSVerify bool     `envconfig:"KAFKA_TLS_INSECURE_SKIP_TLS_VERIFY" default:"true"`
 	TLSPassphrase            string   `envconfig:"KAFKA_TLS_PASSPHRASE"`
+	SASLMechanism            string   `envconfig:"KAFKA_SASL_MECHANISM" default:""`
```
Comment (weeco) [inline options/options.go]: Can you move this up to the other SASL options so that these are grouped together?

**Comment by weeco** [comment]
@jchomat FYI: https://github.com/cloudworkz/kafka-minion/commit/a1642e37455d3075e75831e978ed0edcf29b04cf

**Comment by jchomat** [comment]
Hey @weeco,
Thanks a lot, i'm just back from holidays. i will move on the last version ;)

I've seen that the label host is missing for prometheus filtering when i have multiple cluster.
i've used the prefix_metrics to avoid that but it's not the easiest way for maintain dashboard
does anyone work on this subject?

BR,
Jérémy

**Comment by weeco** [comment]
Take note that I haven't released a new version with the SCRAM support yet. I'd like to fix some other things before releasing a new version. However you are obviously free to use the `master` build which is rebuilt regularly. 

I'd recommend you to add annotations/additional labels for your prometheus job instead. I want to avoid adding multi tenant support in Kafka Minion because that adds a lot complexity (code and configuration) which in my opinion can be avoided by just deploying separate instances for each Kafka cluster with unique prometheus labels attached to these scrapes.

**Comment by jchomat** [comment]
Ok thank for your answer, i will wait for the release and take a look on additionnals label ;)
Deploying multiple exporter is not a problem but just using multiple prefix ;)

**Pull request merged**